test/integration/no_kubernetes_test.go

```diff
@@ -70,6 +71,17 @@ func TestNoKubernetes(t *testing.T) {
 	})
 }
 
+// ErrorStartNoK8sWithVersion expect an error when starting a minikube cluster without kubernetes and with a kubernetes version.
```

Comment:
func name here in comment should match the func

Reply:
Yes, applied suggestion.